packages/teleport/src/dashboard/features/featurePeople.ts

```diff
@@ -14,36 +14,36 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-import * as Icons from 'design/Icon';
-import Account from 'teleport/components/Account';
-import { withFeature, FeatureBase } from 'teleport/components/withFeature';
+import People from 'teleport/dashboard/People';
+import { FeatureBase } from 'teleport/components/withFeature';
 import Ctx from 'teleport/teleportContext';
 import cfg from 'teleport/config';
 
-class AccountFeature extends FeatureBase {
-  Component = withFeature(this)(Account);
+class FeaturePeople extends FeatureBase {
+  Component = People;
 
   getRoute() {
     return {
-      title: 'Account',
-      path: cfg.routes.account,
-      exact: true,
+      title: 'People',
```

Comment:
`People` is ambiguous. I think `Team` is a better word here

Reply:
> People is ambiguous. I think Team is a better word here

we do not have teams but we do have people so it's better to be a bit ambiguous than sending a wrong message.